resources/functionMap.php

```diff
@@ -11967,7 +11967,9 @@
 'stomp_version' => ['string'],
 'StompException::getDetails' => ['string'],
 'StompFrame::__construct' => ['void', 'command='=>'string', 'headers='=>'array', 'body='=>'string'],
+'str_decrement' => ['non-empty-string'],
```

Comment:
Since `str_decrement('1')` returns "0", it is typed as `non-empty-string`. https://3v4l.org/dKARI#v8.3.0